pkg/proxy/winkernel/proxier.go

```diff
@@ -605,6 +683,15 @@ func NewProxier(
 			return nil, fmt.Errorf("source-vip flag not set")
 		}
 
+		if nodeIP.IsUnspecified() {
+			// attempt to get the correct ip address
+			klog.V(2).InfoS("node ip was unspecified.  Attempting to find node ip")
+			nodeIP, err = apiutil.ResolveBindAddress(nodeIP)
+			if err != nil {
+				klog.Warningf("failed to find an ip. You may need set the --bind-address flag")
```

Comment:
do you see a value in moving past this failure with just a warning? Do we expect the networking to work if nodeIP is not available?

Reply:
Yeah, `nodeIP` seems to be referenced later. So would that fail if there is an error here?

Reply:
it would fail later with `F0408 17:17:11.043472 1316 server.go:489] unable to create proxier: unable to create ipv4 proxier: Could not find host mac address for 0.0.0.0, hostname: e2e-wmcow-bpfpl, clusterCIDR : 10.132.0.0/24, nodeIP:0.0.0.0` which is the same behavior.

I was following the best effort pattern from https://github.com/kubernetes/kubernetes/blob/2f753ec4c826895e4ccd3d6bdda2b1ab777ceeb8/cmd/kube-proxy/app/server_others.go#L442. 

If we know it will always fails in this scenario we could exit early.